### DISCUSSION TITLE
陶氏化学 Dow Chemical

### DISCUSSION THREAD

**Discussion by tonyqus** [shnug/jobs#33] [General]
注：以下内容仅供参考，不一定准确

门户网站（dow.com）并不是.NET写的，是Java+AEM实现的。招.NET的部门，主要是DMC和其他几个周边的团队，一些内部系统开发。DMC是隶属于DxC的部门, DxC的全称是Digital Experience Center。

DMC上海怎么说呢，没有什么实质性的项目，基本都是各种无中生有的项目，也可能与传统企业不太需要数字化有关。我记得DMC最拿得出手的IOT项目尽然是食堂信息化项目，我也是醉了。再比如，Azure明明自己有管理门户，隔壁部门就想着再在Azure门户外包一层，搞了个自动建项目的门户，还提供了一些项目模板，意思是你选择好项目模板就能直接建好项目所需的各种资源了，想法是好的，但实操的时候限制的真太多了，各种问题。然后外包这边负责这个项目的架构的同事几乎每天都要给我抱怨，他们在那里瞎搞的各种细节，给我整乐了。

Dow的主要项目架构都是美国人搭的，但有几个美国开发也是半路出家，然后搞出来的东西是有些让人你看不懂的。比如Dow的门户的docker image有80G这么大，我当时都惊呆了，好几次给美国提了建议缩小image，没理我。

另外，Dow上海的网速真的是特别要吐槽的点，慢如龟速（很多时候下载平均速度几十k/秒），性子急的人真不适合在那里呆。

**Comment by jackmingren** [comment]
你给我整乐了

**Reply by tonyqus** [reply]
咋了？

**Reply by jackmingren** [reply]
看到你说的最拿手的是IOT的食堂信息化项目时，想起了我之前在一家传统公司做的项目全死了，只有一个智慧厕所项目还在运行，哈哈。

**Comment by kesshei** [comment]
在成都就好了，可以试试。
